src/components/idleMembers/card/index.tsx

```diff
@@ -1,12 +1,14 @@
-import { FC } from 'react';
+import { FC, useState } from 'react';
 import classNames from '@/components/idleMembers/card/card.module.scss';
 
 type Props = {
   idleMemberUserName: string
 }
 
 const Card: FC<Props> = ({ idleMemberUserName }) => {
-  const imageGenerator = (name: string) => `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${name}/img.png`;
+  const [imageGenerator, setAssigneeProfilePic] = useState(`${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}${idleMemberUserName}/img.png`);
+
+  const contributorImageOnError = () => setAssigneeProfilePic('dummyProfile.png');
```

Comment:
NIT: Can we rename it as `assigneeImageOnError`

Reply:
Can we please get `dummyProfile.png` from the constants file? It will be easier to maintain in the future.
even @abhinav1590 has used `dummyProfile.png`, so moving this to constants will help us. Later on, when we have to change the name of this, we can only change it in one place, rather than changing at all the places where we have hardcoded it.